template.yaml

```diff
@@ -213,6 +213,22 @@ Parameters:
     Description: The (pre-existing) Amazon S3 bucket to store CloudFront access logs in, for example, myawslogbucket.s3.amazonaws.com. Only of use if "CreateCloudFrontDistribution" is set to "true" (the default).
     Type: String
     Default: ""
+  ResourceSuffix:
+    Description: The lambda suffix to use, will be common to lambdas and logs
+    Type: String
+    Default: !Ref "AWS::StackId"
+  LogsDeletionPolicy:
+    Description: Retain Policy for logs when deleting stack
+    Type: String
+    Default: "Retain"
+    AllowedValues:
+      - "Delete"
+      - "Retain"
+      - "RetainExceptOnCreate"
+  LogsRetentionInDays:
+    Description: "Retention in days for logs"
+    Type: Integer
+    Default: 3653
```

Comment:
365?

Reply:
Since there is no retention policy for now, I set to 10y, which sounds reasonable compared to infinite time. I assume users not specify it will prefer having 10y than a too short period.
Use can set/modify retention time as wished

Reply:
yes make sense.